doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -357,21 +344,7 @@ glspl{component}
 
 \end_inset
 
- interact with each other.
- If a 
-\begin_inset ERT
-status open
-
-\begin_layout Plain Layout
-
-
-\backslash
-gls{component}
-\end_layout
-
-\end_inset
-
- calls another 
+ (or bundles) at a time, some internal 
```

Comment:
There is no such thing as an “internal component”. How about “[…] other components that are called during the measured component’s execution may wished to be mocked”.